.all-contributorsrc

```diff
@@ -355,6 +355,17 @@
         "infra",
         "code"
       ]
+    },
+    {
+      "login": "amirebrahimi",
+      "name": "Amir Ebrahimi",
+      "avatar_url": "https://avatars.githubusercontent.com/u/904110?v=4",
+      "profile": "https://github.com/amirebrahimi/",
+      "contributions": [
+        "doc",
+        "test",
+        "code"
+      ]
```

Comment:
See this comment: https://github.com/unitaryfund/mitiq/pull/1484#discussion_r968442938